### ISSUE TITLE
Clothing1M dataset accuracy

### ISSUE THREAD

**Issue by ynu-yangpeng** [ContrastToDivide/C2D#7]
Hello, your work is very interesting! I downloaded your code and attempted to train the model on the clothing1M dataset using the training command you provided (python3 main_clothing1M.py --data_path /path/to/clothing1m --experiment-name selfsup --method selfsup --p_threshold 0.7 --warmup 5 --num_epochs 120). However, I did not achieve the corresponding results. Did I miss something or misconfigure something? Here is my training log:
https://drive.google.com/file/d/1vbc66GHi8qeR3Dnwa3JDAmkPwkOdp2HQ/view?usp=sharing
